beacon_node/network/src/sync/manager.rs

```diff
@@ -1224,27 +1243,26 @@ impl<T: BeaconChainTypes> SyncManager<T> {
 
     /// Handles receiving a response for a range sync request that should have both blocks and
     /// blobs.
-    fn range_block_and_blobs_response(
+    fn on_range_components_response(
         &mut self,
-        id: Id,
+        id: ComponentsByRangeRequestId,
```

Comment:
perhaps rename this to `range_request_id`? 
The `id.id` below is a bit confusing to me.

Reply:
Fixed with 20d97941e